test/testGetPost.py

```diff
@@ -18,24 +18,38 @@ def getRequest(postResponse, url):
 
 class TestCase(unittest.TestCase):
 
-    def testDataValide(self):
+    def testDataValid(self):
         # 1. initialize param. change to others for later use
         url = "http://localhost:80/scenarios"
         
         for fileName in glob(pattern):
-            inputData = json.load(open(fileName)) 
-            # 2. POST request
-            postResponse = requests.post(url, data = json.dumps(inputData), headers = header)
-
-            # 3. GET request
-            queriedData = getRequest(postResponse, url)
-            
-            # 4. validate input data and queried data
-            for key in inputData:
-                # check missing key
-                self.assertTrue(key in queriedData)
-                # check key-value pair match
-                self.assertTrue( inputData[key] == queriedData[key] )
+            with open(fileName) as fp:
+                inputData = json.load(fp) 
+                # 2. POST request
+                postResponse = requests.post(url, data = json.dumps(inputData), headers = header)
+
+                # 3. GET request
+                queriedData = getRequest(postResponse, url)
+                
+                # 4. validate input data and queried data
+                for key in inputData:
+                    # check missing key
+                    self.assertTrue(key in queriedData)
+                    # check key-value pair match
+                    self.assertTrue( inputData[key] == queriedData[key] )
+
+    def testForFailReport(self):
+         # 1. initialize param. change to others for later use
+        url = "http://localhost:80/scenarios"
+        
+        for fileName in glob(pattern):
+            with open(fileName) as fp:
+                inputData = json.load(fp) 
+                # POST request
+                postResponse = requests.post(url, data = json.dumps(inputData), headers = header)
+                # this should be false. intent to fail the test
+                self.assertTrue( postResponse.status_code == 200 )
```

Comment:
Isn't status_code 200 a success rather than failure? The name of this test and the comments are confusing.